src/oas.md

```diff
@@ -4069,7 +4069,12 @@ flows:
 #### Security Requirement Object
 
 Lists the required security schemes to execute this operation.
-The name used for each property MUST correspond to a security scheme declared in the [Security Schemes](#security-scheme-object) under the [Components Object](#components-object).
+
+The name used for each property MUST either correspond to a security scheme declared in the [Security Schemes](#security-scheme-object) under the [Components Object](#components-object), or be the URI of a Security Scheme Object.
+Property names that match the syntax of a component name under the Components Object MUST be treated as a component name.
```

Comment:
I'd prefer
```suggestion
Property names that are the name of a security scheme declared under the Components Object MUST be treated as a component name.
```
"Match the syntax" can be misunderstood as match the regular expression `^[a-zA-Z0-9\.\-_]+$` irrespective of the existence of a security scheme with that name.